### DISCUSSION TITLE
How to delete auth system fields

### DISCUSSION THREAD

**Discussion by jackwellerreal** [pocketbase/pocketbase#1426] [Q&A] [answered]
How would you delete the premade system fields when creating a new collection? I want to create fields with custom options and not use the premade ones.

**Comment by ganigeorgiev** [comment, marked as answer]
You can't remove the `base` collection system fields `id`, `created`, `updated`, nor the `auth` collection system fields `username`, `email`, `emailVisibility`, `verified`. They are always available and used/managed internally.

**Reply by jackwellerreal** [reply]
Oh thanks for replying so fast, may I also ask what the `verified` field is used for?

**Reply by ganigeorgiev** [reply]
`verified` indicates whether the user is "verified" (eg. the user has confirmed their email).
